keycloak_realm_builder/scripts/tasks/realm_tasks/link-idp-realm.yml

```diff
@@ -8,7 +8,7 @@
 
 - name: Set IDP name mapper fact
   set_fact:
-    idp_name_id: { IDIR: idir, GitHub: _github, BCeID: _bceid }
+    idp_name_id: { IDIR: idir, GitHub: _github, BCeID: _bceidbasicbusiness }
```

Comment:
This needs a comment. We made the change because we are restricting "personal" bceids. Very important.

Reply:
Sure will add comment on it